tests/decorators/test_blob.py

```diff
@@ -0,0 +1,57 @@
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License.
+import unittest
+
+from azure.functions import DataType

```

Comment:
nit - directly import from the defining module and not from the top-level module. We the path changes in the future, this should be caught.